Makefile

```diff
@@ -385,6 +385,7 @@ OA_ANSIBLE_BRANCH ?= release-3.10
 define build-cluster-operator-ansible-image #(repo, branch, imagename, tag, clusterapi_playbook)
         cp build/cluster-operator-ansible/playbooks/cluster-api-prep/$5 build/cluster-operator-ansible/playbooks/cluster-api-prep/deploy-cluster-api.yaml
         cp bin/aws-machine-controller build/cluster-operator-ansible/playbooks/cluster-api-prep/files
+        cp bin/cluster-operator build/cluster-operator-ansible/playbooks/cluster-api-prep/files
```

Comment:
This moves the cluster-operator binary into the image as well as the aws-machine-controller binary, so we can transfer it to the masters in devel scenarios. Ultimately the aws machine controller should move into this binary.